### PR TITLE
week3

### PR DESCRIPTION
**Pull request by zehrakopar**

### CODE REVIEW

File: week3/homework/src/index.js [hunk 1]
```diff
@@ -1,3 +1,55 @@
...
+const remove = require('./remove');
+const reset = require('./reset');
+const readTodo = require('./readTodo');
+const markAsDone = require('./markAsDone');
+const markAsNotDone = require('./markAsNotDone');
+
+app.use(express.json());
+
+try {
+  app.get('/todos/:id', (request, response) => {
```
Comment (gajduk) [inline week3/homework/src/index.js]: there should  also be a get todos that returns a list of all todos

File: week3/homework/src/index.js [hunk 2]
```diff
@@ -1,3 +1,55 @@
...
+    readTodo(request.params.id);
+    response.status(200).json({ message: `todo with the id of ${request.params.id} is requested` });
+  });
+
+  app.post('todos/:id/:text', (request, response) => {
```
Comment (gajduk) [inline week3/homework/src/index.js]: this is not the correct format for creating todos. Please review the lecture video. You need to use the body of the requests

File: week3/homework/src/index.js [hunk 3]
```diff
@@ -1,3 +1,55 @@
...
+    add(request.params.id, request.params.text);
+    response.status(201).json({ message: `todo with the id of ${request.params.id} is added` });
+  });
+
+  app.put('todos/:id/:text', (request, response) => {
```
Comment (gajduk) [inline week3/homework/src/index.js]: same here

**Comment by fedefu** [comment]
Thanks for the feedback @gajduk! @zehrakopar can you please implement the feedback?

File: week3/homework/src/index.js (hunk 1 above)
Comment (zehrakopar) [reply week3/homework/src/index.js]: i ve made this changes

**Comment by gajduk** [comment]
Good job.

**Pull request closed without merging**